### ISSUE TITLE
Remove land-ack from the footer of SiteNow websites

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#8355]
We've been asked to remove the land acknowledgement from the footer of all sites as well as any additional places this may be (brand site, UIDS, sitenow docs, dispatch templates).

# Additional Info

This needs to be done prior to Jan 1, 2025.

We wil want to let Jeneane know once this is complete.

Jeneane is working on the comms outreach to the relevant unit. 
```[tasklist]
### Proposed Solution
- [x] Remove land acknowledgement from UIDS footer - https://github.com/uiowa/uids/pull/962
- [x] Remove land acknowledgement from footer in uids_base - https://github.com/uiowa/uiowa/blob/main/docroot/themes/custom/uids_base/templates/uids/footer--socket-menu.twig - may need a cache clear - https://github.com/uiowa/uiowa/pull/8361
- [x] verify regents theme, healthcare split, etc are no longer showing the link - this can be tested once rolled out. Should all be handled by the template listed above
- [x] remove from brand site (Jess and Sondra gave us the ok to make changes) - https://brand.uiowa.edu/applying-brand/web#brand-footer and any other places on brand.uiowa.edu
- [x] Communicate with Dispatch folks about removing this from any templates
- [x] We may need to send comms to app-dev community. We would need to wait for Jeneane to share comms with us
- [x] Send comms to web community (forward Jeneane's message)
- [x] reach out to CLAS marketing/comms team and see if they can handle their sites or let us know if they need assistance - https://sustainability.uiowa.edu/
- [x] Viewbooks - exists on Law Viewbook, Vue project template maybe
- [x] Reach out to catalog.registrar.uiowa.edu about LandAck in their vended solution.
```

**Comment by mjoneill** [comment]
Also, might have impact on guidance we offer on brand site, components in UIDS, sitenow docs, dispatch templates, elsewhere?

**Comment by mjoneill** [comment]
Will also want to check with the Dispatch folks re: templates there.

**Comment by briand44** [comment]
Comment from Michael...

CPH and Libraries have the landack on at least their primary sites…and will need to push a change.  I was expecting them to be covered by the planned comms already + I can followup individually

**Comment by mjoneill** [comment]
Jeneane sent to campus communicators.

I forwarded to:

- App Devs listserv
- UI Web Community listserv

I also forwarded directly to:
- Dan M and Patrick R at CPH
- Ken C, Jose, and Anne B at Libraries
- marketing account at Iowa Law

**Comment by mjoneill** [comment]
I think I have the land ack removed from non-footer sections of the Brand site except for this embed on the [Applying the Brand - Web page](https://brand.uiowa.edu/applying-brand/web): 

<img width="1317" alt="Image" src="https://github.com/user-attachments/assets/60b268a0-8589-4652-9ae6-b5d7b2ed0ce5" />

**Comment by pyrello** [comment]
> I think I have the land ack removed from non-footer sections of the Brand site except for this embed on the [Applying the Brand - Web page](https://brand.uiowa.edu/applying-brand/web):
> 
> <img alt="Image" width="1317" src="https://private-user-images.githubusercontent.com/7144234/398745411-60b268a0-8589-4652-9ae6-b5d7b2ed0ce5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUyMjU5MjIsIm5iZiI6MTczNTIyNTYyMiwicGF0aCI6Ii83MTQ0MjM0LzM5ODc0NTQxMS02MGIyNjhhMC04NTg5LTQ2NTItOWFlNi1iNWQ3YjJlZDBjZTUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIyNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMjZUMTUwNzAyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzI5NWRhOGI1MmQyZTM1YTg4MmYzZDM5MmRhY2MxOGUwODU5MThmMmU2NzUzODcyOWU5ZmQ4N2UwNzk2NWNhZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.1gpNLvPvoZbvZ3K5BxY95Wi_aHsMeeeD1X-c3eOjLUw">

We need to cut a new image for this.

**Comment by mjoneill** [comment]
<visibility>
Before I open the door with the Dispatch folks, I'm confirming with Jeneane that email comms are considered in-scope for the land acknowledgement link and statement removal work.

**Comment by james917** [comment]
> Viewbooks - exists on Law Viewbook, Vue project template maybe

I am going to work on this.

**Comment by mjoneill** [comment]
Visibility: I reached out to CLAS, Catalog, and Dispatch folks.  Catalog uses a vended product and may need to work with the vendor to manage footer links (unsure).

**Comment by briand44** [comment]
Doesn't look like the Regents theme is outputting the Iowa footer links so we should be fine there. - https://wirrc.org/

CCOM split looks fine as well - https://shpep.medicine.uiowa.edu/

**Comment by briand44** [comment]
https://brand.uiowa.edu/applying-brand/web#brand-footer may be the only thing we still need to address that we are aware of.

**Comment by mjoneill** [comment]
> https://brand.uiowa.edu/applying-brand/web#brand-footer may be the only thing we still need to address that we are aware of.

Sweet. 

So, all SNv3, SNv2, and SNv1 sites (custom and standard) should be addressed...plus alternative splits/themes (regents, ccom, etc...)

How should I think about remaining non sitenow D7 sites (like might still exist in CLAS)?

**Comment by briand44** [comment]
> So, all SNv3, SNv2, and SNv1 sites (custom and standard) should be addressed...plus alternative splits/themes (regents, ccom, etc...)
> 
> How should I think about remaining non sitenow D7 sites (like might still exist in CLAS)?

We didn't address SNv1 sites, since I don't believe they ever had the land acknowledgement.

Well, when grooming we had discussed letting CLAS know and then telling them to let us know if they need help. https://sustainability.uiowa.edu was a D7 CLAS site that had it but it looks like they have removed it now.

**Comment by joewhitsitt** [comment]
A release for uids.brand.uiowa.edu still needs to be created right? The change is only showing for /latest

**Comment by pyrello** [comment]
> A release for uids.brand.uiowa.edu still needs to be created right? The change is only showing for /latest

Yep. @james917 and I can address this today.

**Comment by pyrello** [comment]
> > A release for uids.brand.uiowa.edu still needs to be created right? The change is only showing for /latest
> 
> Yep. @james917 and I can address this today.

We are going to work on this on Monday, FYI.

**Comment by james917** [comment]
> > > A release for uids.brand.uiowa.edu still needs to be created right? The change is only showing for /latest
> > 
> > 
> > Yep. [@james917](https://github.com/james917) and I can address this today.
> 
> We are going to work on this on Monday, FYI.

This is done.

**Comment by mjoneill** [comment]
Thanks for everyone's patience on the updating the image on the brand site. 

Done.  

<img width="1326" alt="Image" src="https://github.com/user-attachments/assets/be358da9-05a4-43d9-ae6b-4d81f362359f" />

**Comment by briand44** [comment]
@mjoneill are you comfortable with us closing this issue as complete now that the footer image has been updated on the Brand site?

**Comment by mjoneill** [comment]
> @mjoneill are you comfortable with us closing this issue as complete now that the footer image has been updated on the Brand site?

Yes.  This scoped work is done. Note that some of the work was _asking_ campus partners to update their websites, so while this issue may be closed...there will likely be additional outreach investments to make, especially given that much of the outreach is hitting people's inboxes while they are on vacation.  Those can either be off board as not the work I'm flowing to the team, or captured in a separate card in the future.

Example: 

<img width="1264" alt="image" src="https://github.com/user-attachments/assets/5181f5b4-c76f-4975-a17b-1b2be6e7ec01" />
